src-env (msdb)/.env/dev-use1/.env

```diff
@@ -0,0 +1,25 @@
+REGION=USE1 #East USA 1
```

Comment:
Did you added what we agreed to the readme? i asked you to add the naming convesion of the env:
camel case, sections separate by "_"

Reply:
Yes I added